tests/integration/test_subscriptions_cli.py

```diff
@@ -292,7 +292,7 @@ def test_request_catalog_success(invoke, geom_geojson):
         "parameters": {
             "geometry": geom_geojson,
             "start_time": "2021-03-01T00:00:00Z",
-            "item_types": ["PSScene"],
```

Comment:
Removed the list around `item_types` in all tests as it is now declared as a `click.Choice()` (see https://github.com/planetlabs/planet-client-python/pull/905/files#diff-c5c9e91e4feb36110525dcd81b6203ded54729974229d5625a906e09b9697936)

Reply:
According to the subscriptions createsubscription [docs](https://developers.planet.com/docs/subscriptions/reference/#tag/subscriptions/operation/createSubscription), this entry must be an array of strings, so I don't think we want to change the test here, I think we want to change the function it's testing
![image](https://user-images.githubusercontent.com/2984664/230154115-1586eaf6-34a3-4050-99bb-6729728d2bc1.png)

Reply:
looking below, this applies to everywhere in the tests where the array has been changed to a string.